include/deal.II/fe/fe_interface_values.h

```diff
@@ -2125,6 +2827,375 @@ namespace FEInterfaceViews
 
 
 
+  template <int dim, int spacedim>
+  template <class InputVector>
+  void
+  Scalar<dim, spacedim>::get_function_values_from_local_dof_values(
+    const bool         here_or_there,
+    const InputVector &local_dof_values,
+    std::vector<solution_value_type<typename InputVector::value_type>> &values)
+    const
+  {
+    Assert(values.size() == this->fe_interface->n_quadrature_points,
+           ExcNotImplemented());
```

Comment:
```suggestion
    AssertDimension(values.size(), this->fe_interface->n_quadrature_points);
```
Both here and in the other `get_function_*_from_local_dof_values()`.